CRM/Mailing/BAO/Mailing.php

```diff
@@ -1067,15 +1067,13 @@ public function getVerpAndUrlsAndHeaders($job_id, $event_queue_id, $hash, $email
    * @param string $fromEmail
    *   Email address of who is forwarding it.
    *
-   * @param null $replyToEmail
-   *
    * @return Mail_mime               The mail object
    */
   public function compose(
     $job_id, $event_queue_id, $hash, $contactId,
     $email, &$recipient, $test,
     $contactDetails, &$attachments, $isForward = FALSE,
-    $fromEmail = NULL, $replyToEmail = NULL
+    $fromEmail = NULL
```

Comment:
May break for 
https://github.com/civicrm/civicrm-core/blob/master/CRM/Mailing/BAO/MailingJob.php#L604 ?

Reply:
Okay may not require if https://github.com/civicrm/civicrm-core/pull/29149 is merged